package.json

```diff
@@ -35,6 +35,7 @@
     "test": "cypress run",
     "cypress:open": "cypress open",
     "codegen:dbc:gateway": "CODEGEN_SCHEMA_AGENCY=100200 CODEGEN_SCHEMA_PROFILE=opac graphql-codegen --config dbc-gateway.codegen.yml",
+    "codegen:rest-services": "orval",
```

Comment:
Add `--config <FILE>` to use the one that matches this client.

Reply:
I will resolve this based on the comment above.